executor/show.go

```diff
@@ -571,6 +571,9 @@ func (e *ShowExec) fetchShowCreateTable() error {
 	var hasAutoIncID bool
 	for i, col := range tb.Cols() {
 		fmt.Fprintf(&buf, "  %s %s", escape(col.Name, sqlMode), col.GetTypeDesc())
+		if col.Charset != "binary" {
```

Comment:
why don't we print the "binary" charset and collation?

Reply:
Because MySQL didn't do it either, and it will broken lots of test case...

Reply:
OK, this behavior is confirmed in a mysql server on my laptop:
```sql
MySQL(root@localhost:test) > create table t(a char(10) character set binary, b char(20) character set utf8mb4);
Query OK, 0 rows affected (0.03 sec)

MySQL(root@localhost:test) > show create table t\G
*************************** 1. row ***************************
       Table: t
Create Table: CREATE TABLE `t` (
  `a` binary(10) DEFAULT NULL,
  `b` char(20) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
1 row in set (0.00 sec)
```